documentation/domains/Domain.json

```diff
@@ -496,6 +496,10 @@
           "description": "A brief CamelCase message indicating details about why the domain is in this state.",
           "type": "string"
         },
+        "rolling": {
+          "description": "True while pods have been intentionally deleted and have not yet returned to the ready state",
```

Comment:
We need a statement about the default value, which I assume is false, for when the field is not populated.